components/BakeryMenu.js

```diff
@@ -13,6 +19,12 @@ export default function BakeryMenu({ image, name, ingredients, weight, cost }) {
       <ImageWrapper>
         <Image src={image} layout="fixed" width={135} height={135} />
       </ImageWrapper>
+      <HiddenOptions
+        showHiddenOptions={showHiddenOptions}
+        style={{ display: showHiddenOptions ? ' ' : 'none' }}
+      >
+        <AddToBasketButton>Add to Basket</AddToBasketButton>
+      </HiddenOptions>
```

Comment:
This could be done easier. You don't need an extra styled component for that.
You could try this:
```suggestion      
        { showHiddenOptions && <AddToBasketButton>Add to Basket</AddToBasketButton>}
```

Reply:
Thank you @manusmd for looking this over and for the suggestion, I just made the changes 🙏